lib/matplotlib/lines.py

```diff
@@ -534,7 +534,7 @@ def set_fillstyle(self, fs):
 
             For examples see :ref:`marker_fill_styles`.
         """
-        self._marker.set_fillstyle(fs)
+        self.set_marker(MarkerStyle(self._marker.get_marker(), fs))
```

Comment:
I guess we never test `set_fillstyle`?  Can we use this as the opportunity to do so?

Reply:
I'd like to have this deprecation be part of 3.4 and I don't know if I have the bandwidth to add tests soon. Therefore, I'd rather do this in a later PR.